WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.swift

```diff
@@ -251,6 +258,43 @@ class NotificationDetailsViewController: UIViewController, NoResultsViewHost {
         previousNavigationButton.accessibilityLabel = NSLocalizedString("Previous notification", comment: "Accessibility label for the previous notification button")
         nextNavigationButton.accessibilityLabel = NSLocalizedString("Next notification", comment: "Accessibility label for the next notification button")
     }
+
+    fileprivate func setupShareFooterView() {
+        guard let contentUrl = note.url else {
+            return
+        }
+        var shareKind: ShareFooterView.Kind
+        switch note.kind {
+        case .like:
+            shareKind = .post
+        case .commentLike:
+            shareKind = .comment
+        case .follow:
+            shareKind = .blog
+        default:
+            return
+        }
+        let shareFooterView = ShareFooterView(kind: shareKind) {
+            let activityViewController = UIActivityViewController(activityItems: [contentUrl as Any], applicationActivities: nil)
+            activityViewController.popoverPresentationController?.sourceView = self.stackView
```

Comment:
[Suggestion] The activity view will appear like this on iPad when the `sourceView` is `self.stackView`:

![CleanShot 2024-05-31 at 19 44 26@2x](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/3c73d885-4c1e-4f16-a7fd-5957f633c654)

I suggest to set `self.shareFooterView` instead.